jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -909,14 +913,14 @@ pipeline {
                     if (params.PUBLISH_NOTIFICATION) {
                         publishNotification(
                             icon: ':warning:',
-                            message: buildFailureMessage(),
+                            message: buildMessage(search: 'Error building'),
                             credentialsId: 'jenkins-build-notice-webhook',
                             manifest: "${INPUT_MANIFEST}"
                         )
                     }
                     if (params.CREATE_GITHUB_ISSUE) {
                         createGithubIssue(
-                            message: buildFailureMessage()
+                            message: buildMessage(search: 'Error building'),
```

Comment:
Can you also another arg `inputManifestPath: "manifests/$INPUT_MANIFEST"` similar to https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/opensearch/distribution-build.jenkinsfile#L962-L967

Reply:
Ya it was using an older version of the library, the new one should be as 
```
createBuildFailureGithubIssue(
    message: buildMessage(search: 'Error building'),
    inputManifestPath: "manifests/$INPUT_MANIFEST"
)
  ```

Reply:
Thanks, updated @gaiksaya.